src/client/opamArg.ml

```diff
@@ -1020,6 +1021,15 @@ let config =
     "Without argument, prints a documented list of all available variables. With \
      $(i,PACKAGE), lists all the variables available for these packages. Use \
      $(i,-) to include global configuration variables for this switch.";
+    ["set"]     , `set     , ["VAR";"VALUE"],
+    "Set the given global opam variable for the current switch. Warning: \
+     changing a configured path will not move any files! This command does \
+     not perform any variable expansion.";
+    ["unset"]   , `unset     , ["VAR"],
+    "Unset the given global opam variable for the current switch. Warning: \
+     unsetting built-in configuration variables can cause problems!";
+    ["expand"]  , `expand  , ["STRING"],
```

Comment:
hmm, `eval` would be better, but in my brain, that connects with something that is going to be executed... so I was afraid of a confusion with `exec`. We also have `eval $(opam config env)` all around the place so I'm trying to avoid it.